composer.json

```diff
@@ -16,17 +16,17 @@
         }
     ],
     "require": {
-        "php":                          "^5.5|^7.0",
```

Comment:
Technically there is no need to drop support for older PHP versions, and same goes for older Symfony versions. Doing this _would_ require a new major version.

Reply:
You're right, I'll re-add 5.5.9 @sstok 👍